spec/calendar.html

```diff
@@ -582,17 +603,17 @@ <h1>
           1. NOTE: This is the last week of the previous year.
           1. Let _dayOfJan1st_ be ToISODayOfWeek(_year_, 1, 1).
           1. If _dayOfJan1st_ is _friday_, then
-            1. Return _maxWeekNumber_.
+            1. Return { [[Week]]: _maxWeekNumber_, [[Year]]: _year_ - 1 }.
```

Comment:
```suggestion
            1. Return the Record { [[Week]]: _maxWeekNumber_, [[Year]]: _year_ - 1 }.
```
(or "the WeekYear Record {" if you take the above suggestion). same throughout